site/content/docs/main/plugin-release-instructions.md

```diff
@@ -9,9 +9,21 @@ Container images are built via a CI job on git push.
 
 Plugins the Velero core team is responsible include all those listed in [the Velero-supported providers list](supported-providers.md) _except_ the vSphere plugin.
 
+
+## Steps
+### Open a PR to prepare the repo
 1. Update the README.md file to update the compatibility matrix and `velero install` instructions with the expected version number and open a PR.
-  1. Determining the version number is based on semantic versioning and whether the plugin uses any newly introduced, changed, or removed methods or variables from Velero.
-1. Once the updated README.md PR is merged, checkout the upstream `main` branch. `git checkout upstream/main`.
+1. Determining the version number is based on semantic versioning and whether the plugin uses any newly introduced, changed, or removed methods or variables from Velero.
+2. Roll all unreleased changelogs into a new `CHANGELOG-v<version>.md` file and delete the content of the `unreleased` folder. Edit the new changelog file as needed.
+### Tag
+1. Once the PR is merged, checkout the upstream `main` branch. Your local upstream might be named `upstream` or `origin`, so use this command: `git checkout <upstream-name>/main`.
 1. Tag the git version - `git tag v<version>`.
-1. Push the git tag - `git push --tags <upstream remote name>`
-1. Wait for the container images to build
+1. Push the git tag - `git push --tags <upstream-name>` to trigger the image build.
+2. Wait for the container images to build. You may check the progress of the GH action that triggers the image build at `https://github.com/vmware-tanzu/<plugin-name>/actions`
+3. Verify that an image with the new tag is available at `https://hub.docker.com/repository/docker/velero/<plugin-name>/`.
+4. Run the Velero e2e tests using the new image. Until it is made configurable, you will have to edit the [plugin version][1] in the test.
```

Comment:
On this section: https://deploy-preview-3591--velero.netlify.app/docs/v1.6/release-instructions/#plugins, we link to the test instructions. Would this be sufficient?

Reply:
yes, i think that link would work. It might be nice to have it on the plugin release instruction page, but if you mean to just have the like on the release instructions page, that would work as well.

Reply:
You are right, it doesn't hurt to include a link here and it's much better. Adding..